sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -336,23 +340,33 @@ export class OfferingValidationModel {
   @Min(0, {
     message: getMinFormatMessage(userFriendlyNames.actualTuitionCosts),
   })
-  @Max(MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE, {
+  @IsNumber(currencyNumberOptions, {
+    message: getCurrencyFormatMessage(userFriendlyNames.actualTuitionCosts),
+  })
+  @Max(MAX_ALLOWED_OFFERING_AMOUNT, {
+    context: ValidationContext.CreateWarning(
+      OfferingValidationWarnings.OfferingCostExceedMaximum,
+    ),
+  })
+  @Validate(CustomMaxLength, {
```

Comment:
IMO the name custom max length seems to be very generic even though it is part of offering custom validators. And also I feel `maxValue` would sound better at the end than `maxLength` in this case

Reply:
mm, I believe it is a generic value, for db safety